### PR TITLE
Add latexpdf support for Greek as main language (via xelatex use)

### PR DESCRIPTION
**Pull request by jfbu**
Closes: #6710

### Feature or Bugfix
- Bugfix

### CODE REVIEW

**Comment by jfbu** [comment]
I have squashed the commits. Python 3.6 mypy testing fails for reasons unrelated to this PR.

File: doc/usage/configuration.rst [hunk 1]
```diff
@@ -1841,7 +1841,17 @@ These options influence LaTeX output.
...
-   the default is the GNU FreeFont which covers well Latin, Cyrillic and Greek.
+   the default is the GNU FreeFont which covers well Latin, Cyrillic and
+   Greek.
+
+   .. versionchanged:: 2.1.0
+
+      Use ``xelatex`` (and LaTeX package ``xeCJK``) by default for Chinese
+      documents.
+
+   .. versionchanged:: 2.2.1
```
Comment (tk0miya) [inline doc/usage/configuration.rst:1852]: Now, we are using x.y.Z branch for urgent bugfix release. I think 2.0 branch is appropriate for this fix.

File: sphinx/builders/latex/__init__.py [hunk 2]
```diff
@@ -418,6 +418,8 @@ def default_latex_engine(config: Config) -> str:
         return 'platex'
     elif (config.language or '').startswith('zh'):
         return 'xelatex'
+    elif (config.language or '') == 'el':
```
Comment (tk0miya) [inline sphinx/builders/latex/__init__.py]: `or ''` is not needed for this case. It is useful to call methods of string object even if `config.language` is None. But this line simply compares with string.

**Review by tk0miya — CHANGES_REQUESTED** [review]
LGTM with nits.

File: doc/usage/configuration.rst (hunk 1 above)
Comment (tk0miya) [reply doc/usage/configuration.rst:1852]: Ah, sorry. I just read #6710 and understand this is a bug. Okay, go with 2.2.1.

**Comment by tk0miya** [comment]
>I have squashed the commits. Python 3.6 mypy testing fails for reasons unrelated to this PR.

I fixed the error in latest 2.2.1 branch today. Please merge it into this. Then error will go away.

File: sphinx/builders/latex/__init__.py (hunk 2 above)
Comment (jfbu) [reply sphinx/builders/latex/__init__.py]: blindly copying pasting is fine, but thinking is better, thanks for the explanation :) will fix

**Comment by jfbu** [comment]
Thanks for reviewing @tk0miya now merging into 2.2.1

**Review by sulaeman512 — APPROVED** [review]
To android ku

**Pull request merged**